projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -367,10 +369,19 @@ const IssueScreenWithPath = React.memo(
                 return (
                     <>
                         <PreviewReloadButton
-                            onPress={async () => {
+                            onPress={() => {
+                                if (isProof) {
+                                    deleteIssueFiles()
```

Comment:
Would be cleaner here to use `async await` with a `try... catch` and then use a `finally` for the restart that always needs to happen